ballerina/tests/call-procedures-test.bal

```diff
@@ -191,7 +192,24 @@ function testMoneyProcedureCall() returns error? {
     };
     test:assertEquals(check queryProcedureClient(query, proceduresDb, MoneyProcedureRecord), expectedDataRow,
                       "Money Call procedure insert and query did not match.");
+}
+
+@test:Config {
+    groups: ["procedures"],
+    dependsOn: [testMoneyProcedureCall]
+}
+function testTimestamptzRetrieval() returns error? {
+    string datetimetz = "2021-07-21T19:14:51.00+01:30";
+    time:Civil civil = check time:civilFromString(datetimetz);
+
+    int rowId = 2;
+    sql:TimestampWithTimezoneOutParameter datetimetzOutValue = new;
+
+    sql:ParameterizedCallQuery sqlQuery = `{call DateTimeOutProcedure (${rowId}, ${datetimetzOutValue})}`;
+    sql:ProcedureCallResult result = check callProcedure(sqlQuery, proceduresDb);
```

Comment:
NullPointerException may occur due to not passing `rowTypes` array.

Reply:
The `rowTypes` array is a default parameter within the `[callProcedure](https://github.com/ballerina-platform/module-ballerinax-mssql/blob/5ca6c985b5e219b1c11844d03c40944a752aacf0/ballerina/tests/call-procedures-test.bal#L230)` method. 

In any case, the NullPointerException still occurs with the `rowTypes` array passed into the method.